### ISSUE TITLE
bad request "OPTIONS"

### ISSUE THREAD

**Issue by erg** [slavapestov/factor#1433]
```
[2015-08-06T16:52:11.850450Z] NOTICE handle-client: { T{ duplex-stream f ~decoder~ ~encoder~ } T{ inet4 f "61.53.122.2" 58335 } T{ inet4 f "0.0.0.0" 8080 } }
[2015-08-06T16:52:11.850797Z] NOTICE accepted-connection: remote: 61.53.122.2:58335, local: 0.0.0.0:8080
[2015-08-06T16:52:11.853206Z] ERROR do-request: Fall-through in case
[--------------------] ERROR do-request: object "OPTIONS"
```

**Comment by erg** [comment]
related to #1431

**Comment by mrjbq7** [comment]
Fixed by 2bb93f1b721f34acf8a9e9bb0c9ba412a8e2140f.
